libraries/RC_Channel/RC_Channel.h

```diff
@@ -380,7 +380,7 @@ class RC_Channels {
     void reset_mode_switch();
     virtual void read_mode_switch();
 
-    // has_valid_input should be pure-virtual when Plane is converted
+    virtual bool in_rc_failsafe() const { return true; };
```

Comment:
can this be pure virtual?

Reply:
I tried the pure-virtual thing. It didn't work out. Too much stuff to be reverted when we move the failsafe handling up.